buffer/src/main/java/io/netty/buffer/PoolThreadCache.java

```diff
@@ -368,7 +369,7 @@ protected abstract void initBuf(PoolChunk<T> chunk, ByteBuffer nioBuffer, long h
         @SuppressWarnings("unchecked")
         public final boolean add(PoolChunk<T> chunk, ByteBuffer nioBuffer, long handle, int normCapacity) {
             Entry<T> entry = newEntry(chunk, nioBuffer, handle, normCapacity);
-            boolean queued = queue.offer(entry);
+            boolean queued = queue.relaxedOffer(entry);
```

Comment:
relaxed offer on mpsc doesn't make much difference https://github.com/JCTools/JCTools/blob/master/jctools-core/src/main/java/org/jctools/queues/MpscArrayQueue.java#L456

while `relaxedPoll` would save it to spin awaiting the elements (and breaking linearizability if the queue);

I don't suggest to go there unless there's some evidence of pathologic cases which can by fixing by this change...

Reply:
> relaxed offer on mpsc doesn't make much difference

True for `relaxedOffer`.

About the `relaxedPoll` part, I may try some benchmark if possible.

Reply:
@laosijikaichele any update ?

Reply:
please ping me once you are ready